solr/core/src/test/org/apache/solr/cloud/TestRSAKeyPair.java

```diff
@@ -36,26 +38,29 @@ public void testGenKeyPair() throws Exception {
   public void testReadKeysFromDisk() throws Exception {
     URL privateKey = getClass().getClassLoader().getResource("cryptokeys/priv_key512_pkcs8.pem");
     URL publicKey = getClass().getClassLoader().getResource("cryptokeys/pub_key512.der");
-
+    assertNotNull(privateKey);
+    assertNotNull(publicKey);
     testRoundTrip(new CryptoKeys.RSAKeyPair(privateKey, publicKey));
   }
 
   private void testRoundTrip(CryptoKeys.RSAKeyPair kp) throws Exception {
-    final byte[] plaintext = new byte[random().nextInt(64)];
-    random().nextBytes(plaintext);
```

Comment:
random().nextBytes() would end up with null bytes and that would cause issues. I switched this to generate a regular string with no null bytes.